src/Swarm/Util/Parse.hs

```diff
@@ -17,3 +21,6 @@ fully sc p = sc *> p <* eof
 --   ensuring that the parser extends all the way to eof.
 fullyMaybe :: (MonadParsec e s f) => f () -> f a -> f (Maybe a)
 fullyMaybe sc = fully sc . optional
+
+sepByNE :: (MonadPlus m) => m a -> m sep -> m (NonEmpty a)
```

Comment:
We should just use `sepBy1` from https://hackage.haskell.org/package/parser-combinators-1.3.0/docs/Control-Monad-Combinators-NonEmpty.html .